package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "cross-env NODE_ENV=development nyc mocha --require @babel/register ./src/__tests__/*.js",
+    "test": "cross-env NODE_ENV=development nyc mocha --require @babel/register --require @babel/polyfill -r dotenv/config ./src/__tests__/*.js --exit",
```

Comment:
what does the `dotenv` config do?

Reply:
`-r dotenv/config ` it makes the `dotenv.config()` environment available in our test environment

Reply:
ok good.